server/graphql/Cargo.toml

```diff
@@ -30,7 +30,7 @@ async-graphql = { version = "3.0.35", features = ["dataloader", "chrono", "log"]
 async-graphql-actix-web = "3.0.35"
 async-trait = "0.1.30"
 chrono = { version = "0.4", features = ["serde"] }
-reqwest = { version = "0.11", features = ["json"] } 
+reqwest = { version = "0.11.10", features = ["json"] } 
```

Comment:
as per main comment, notices two versions used in the project, so picked the latest (and updated other tomls)